src/api/config/routes.rb

```diff
@@ -31,7 +31,7 @@ def self.public_or_about_path?(request)
     arch:       %r{[^\/]*},
     binary:     %r{[^\/]*},
     filename:   %r{[^\/]*},
-    id:         %r{\d*},
+    id:         %r{[^\/]*},
```

Comment:
Shouldn't we stick to the number as id? And if we don't, you might want to update line 384
```
    delete 'person/:login/token/:id' => 'person#delete_token', constraints: { id: %r{[^\/]*}, login: %r{[^\/]*} } 
```